### ISSUE TITLE
Turtle* writer?

### ISSUE THREAD

**Issue by lapidus** [blake-regalia/graphy.js#46]
I am trying to create a test dataset for Stardog using the (somewhat agreed upon?) Turtle* (Turtle-star) syntax:

```
<< :Alice :worksFor :ACME >> :role :CEO ;  
                             :since 2010 ;
                             :probability 0.8 ;
                             :source <http://example.com/news> . 
```

> https://www.stardog.com/blog/property-graphs-meet-stardog/
> 
> When writing and querying edge properties, an extended RDF and SPARQL syntax is needed. The RDF*/SPARQL* proposal defines an extension to Turtle syntax and the above example would look as follows:
> 
> This syntax works well most of the time but various Turtle shortcuts such as implicit bnodes ([]), predicate lists (;), and object lists (,) cannot be used inside the << >> patterns. This makes the syntax more verbose since the same subject has multiple outgoing edges, some of which have properties and some of which do not. To address this issue we support an alternative syntax in Stardog that puts edge properties next to the predicates.

Is there a way to output something like this with Graphy? (Either natively or via some temporary workaround?)

ps. Lovely work with graphy.js overall, big big thanks ...

**Comment by blake-regalia** [comment]
Yes, the upcoming v5 supports serializing to Turtle-star. It also handles reifying triples in subject and object position when writing to Turtle 1.1 for backwards-compatibility.

**Comment by lapidus** [comment]
Brilliant! I’ll take it for a spin on a 3 GB conversion operation when alpha is out :)
